generate_outputs.py

```diff
@@ -0,0 +1,190 @@
+    parser.add_argument("individuals_json_output_path", metavar="individuals-json-output-path",
+                        help="Path to a JSONL file to write the TracedData associated with the individuals analysis file")
+    parser.add_argument("icr_output_dir", metavar="icr-output-dir",
+                        help="Directory to write CSV files to, each containing 200 messages and message ids for use " 
+                             "in inter-code reliability evaluation"),
+    parser.add_argument("coded_dir_path", metavar="coded-dir-path",
+                        help="Directory to write coded Coda files to")
+    parser.add_argument("csv_by_message_output_path", metavar="csv-by-message-output-path",
+                        help="Analysis dataset where messages are the unit for analysis (i.e. one message per row)")
+    parser.add_argument("csv_by_individual_output_path", metavar="csv-by-individual-output-path",
+                        help="Analysis dataset where respondents are the unit for analysis (i.e. one respondent "
+                             "per row, with all their messages joined into a single cell)")
+    parser.add_argument("production_csv_output_path", metavar="production-csv-output-path",
+                        help="Path to a CSV file to write raw message and demographic responses to, for use in "
+                             "radio show production"),
+
+    args = parser.parse_args()
+
+    csv_by_message_drive_path = None
+    csv_by_individual_drive_path = None
+    production_csv_drive_path = None
+
+    user = args.user
+    google_cloud_credentials_file_path = args.google_cloud_credentials_file_path
+    pipeline_configuration_file_path = args.pipeline_configuration_file_path
+
+    raw_data_dir = args.raw_data_dir
+    prev_coded_dir_path = args.prev_coded_dir_path
+
+    messages_json_output_path = args.messages_json_output_path
+    individuals_json_output_path = args.individuals_json_output_path
+    icr_output_dir = args.icr_output_dir
+    coded_dir_path = args.coded_dir_path
+    csv_by_message_output_path = args.csv_by_message_output_path
+    csv_by_individual_output_path = args.csv_by_individual_output_path
+    production_csv_output_path = args.production_csv_output_path
+
+    # Load the pipeline configuration file
+    log.info("Loading Pipeline Configuration File...")
+    with open(pipeline_configuration_file_path) as f:
+        pipeline_configuration = PipelineConfiguration.from_configuration_file(f)
+
+    if pipeline_configuration.drive_upload is not None:
+        log.info(f"Downloading Google Drive service account credentials...")
+        credentials_info = json.loads(google_cloud_utils.download_blob_to_string(
+            google_cloud_credentials_file_path, pipeline_configuration.drive_upload.drive_credentials_file_url))
+        drive_client_wrapper.init_client_from_info(credentials_info)
+
+    # Load the input datasets
```

Comment:
I think we should move this function to the lib. What do you think?

Reply:
agreed, thanks for the suggestion :)

Reply:
I'll do this in a future PR. Tracking this in #13.